### PR TITLE
Test for spawning remote actors

### PR DESCRIPTION
**Pull request by tomliversidge**

### CODE REVIEW

File: tests/Proto.Remote.Tests/RemoteTests.cs [hunk 1]
```diff
@@ -38,9 +38,25 @@ public class RemoteTests
...
+        [Fact]
+        public async void CanSpawnRemoteActor()
+        {
+            var props = Actor.FromProducer(() => new LocalActor());
+            var pid = Actor.Spawn(props);
+            var remoteActorName = Guid.NewGuid().ToString();
+            var remoteActor = await Remote.SpawnNamedAsync("127.0.0.1:12000", remoteActorName, "remote", TimeSpan.FromSeconds(5));
+            await remoteActor.RequestAsync<Start>(new StartRemote { Sender = pid }, TimeSpan.FromMilliseconds(2500));
+            remoteActor.Tell(new Ping());
+            await Task.Delay(TimeSpan.FromSeconds(2));
```
Comment (rogeralsing) [inline tests/Proto.Remote.Tests/RemoteTests.cs]: why do we need the delay?

**Comment by rogeralsing** [comment]
Great!

File: tests/Proto.Remote.Tests/RemoteTests.cs (hunk 1 above)
Comment (tomliversidge) [inline tests/Proto.Remote.Tests/RemoteTests.cs]: It's to wait for the remoteActor.Tell to have time to complete. I guess a better test would be to ReceiveAsync on the remote actor...

Comment (rogeralsing) [inline tests/Proto.Remote.Tests/RemoteTests.cs]: Agree, making it use ReceiveAsync would be better and will fail less often on the build server

Comment (tomliversidge) [inline tests/Proto.Remote.Tests/RemoteTests.cs]: https://github.com/AsynkronIT/protoactor-dotnet/pull/158

**Pull request merged**